etp-backend/src/main/clj/solita/etp/service/energiatodistus_search.clj

```diff
@@ -117,21 +118,28 @@
     #(expression-seq->sql "and" predicate-expression->sql %)
     where))
 
-(def blank? (some-fn nil? empty?))
+(defn whoami->sql [{:keys [id] :as whoami}]
+  (cond
+    (rooli-service/paakayttaja? whoami) ["energiatodistus.tila_id <> ?" 0]
```

Comment:
luulen että tämä kannattaa olla vakio kyselyssä eli ei bind parametri. Kyselyn optimointi voi olla näin tehokkaampaa - kyselyn ei tarvitse olla ainakaan tämän parametrin suhteen bind-aware